app/renderer/js/notification/darwin-notifications.ts

```diff
@@ -103,8 +92,8 @@ class DarwinNotification {
 		customReply(response);
 	}
 
-	// Method specific to notification api
-	// used by zulip
+	// Method specific to Zulip's notification module, not the package used to create the same.
+	// If at all the need arises to migrate to another API (or npm package), DO NOT REMOVE THIS.
```

Comment:
I don’t understand this new comment. [`Notification.close()`](https://developer.mozilla.org/en-US/docs/Web/API/Notification/close) is part of the standard `Notification` API. It’s not Zulip-specific.

Reply:
The previous comment got me believing that this was specific to `node-mac-notifier`. So I tried to refine it. I'll remodel it.